docs/tutorial/dbcon.rst

```diff
@@ -8,7 +8,7 @@ You now have a function for establishing a database connection with
 closing database connections all the time is very inefficient, so you will
 need to keep it around for longer.  Because database connections
 encapsulate a transaction, you will need to make sure that only one
-request at a time uses the connection. An elegant way to do this is by
```

Comment:
@untitaker I'd like you to take a second look at this. I think the changes are related enough that there is not really a need to file a separate PR. `flaskr` and `minitwit` will be/are structured exactly the same way. Therefore the documentation is really relevant to both examples. Most of the changes are about testing, so it's not just superficial grammar stuff. (I'm not being lazy, I just don't agree that they are unrelated :P)